datafusion/core/src/dataframe/parquet.rs

```diff
@@ -246,4 +246,72 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn roundtrip_parquet_with_encryption() -> Result<()> {
```

Comment:
I wonder why this isn't in core/tests as well 🤔   (I see you are just following the existing pattern, I just noticed this while reviewing this PR)

Reply:
I'm happy to move it if you think we should. As you note, I am following the pattern of the existing tests but it may fit better elsewhere.

Reply:
Maybe we can move it in a follow on PR.  -- I would sort of expect the tests to be in  https://github.com/apache/datafusion/tree/main/datafusion/core/tests/dataframe

Perhaps in a file named `parquet.rs`